src/Models/BlogModel.php

```diff
@@ -15,32 +18,36 @@ public function __construct($db)
         $this->db = $db;
     }
 
-    public function GetAllEntries(): bool
+    public function CreateNewEntry($blogPost): bool
     {
-        $query = $this->db->prepare("SELECT `*` FROM `blog-posts` WHERE deleted=0");
-        $query->execute();
-        $Blog = $query->fetchAll();
-        return $Blog;
+        $query = $this->db->prepare('INSERT INTO `blog-posts` (`title`, `author`, `date`, `post`)
+                                        VALUES (:title , :author, :date, :post)');
+        return $queryCheck = $query->execute($blogPost);
     }
 
-    public function CreateNewEntry($BlogPost): bool
+    public function ReadAllEntries(): array
     {
-        $query = $this->db->prepare("INSERT INTO `blog-posts` (`title`, `author`, `date`, `post`) VALUE (:title, :author, :date, :post)");
-        $AddNewEntry = $query->execute($BlogPost);
-        return $AddNewEntry;
+        $query = $this->db->query('SELECT `title`, `author`, `date`, `post`, `GUID`
+                                    FROM `blog-posts` 
+                                    WHERE `deleted` = 0;');
+         return $query = $query->fetchAll();
     }
 
-    public function EditEntry($EditEntry): bool
+    public function UpdateEntry($editEntry): bool
     {
-        $query = $this->db->prepare("SELECT `title` FROM `blog-posts` UPDATE (:title, :Date, :post)");
-        $UpdatedEntry = $query->execute($EditEntry);
-        return $UpdatedEntry;
+        $query = $this->db->prepare('UPDATE `blog-posts`
+                                        set `title` = :title,
+                                            `date` = :date,
+                                            `post` = :post 
+                                        WHERE `GUID` = :GUID;');
+        return $updatedEntry = $query->execute($editEntry);

```

Comment:
again there is no need to store the return into $updatedEntry, just return the final query that gives you what you want